doc/howto/instances_access_files.md

```diff
@@ -0,0 +1,91 @@
+(instances-access-files)=
+# How to access files in an instance
+
+You can access the files from your instance and edit them, delete them, pull them to or push them from the host machine, or mount a file system from the instance to the host machine.
+
+## Edit instance files
+
+To edit an instance file from your host machine, enter the following command:
+
+    lxc file edit <instance_name>/<path_to_file>
+
+For example, to edit the `/etc/hosts` file in the instance, enter the following command:
+
+    lxc file edit my-container/etc/hosts
+
+```{note}
+The file must already exist on the instance.
+You cannot use the `edit` command to create a file on the instance.
+```
+
+## Delete files from the instance
+
+To delete a file from your instance, enter the following command:
+
+    lxc file delete <instance_name>/<path_to_file>
+
+## Pull files from the instance to the host machine
```

Comment:
Lets change all instances of "host machine" to "local machine" to avoid implying that you have to run the `lxc` command on the instance's host machine.